html/themes/custom/common_design_subtheme/components/rw-social-media-links/rw-social-media-links.css

```diff
@@ -0,0 +1,45 @@
+/**
+ * Social media links.
+ *
+ * Block with the a list of links to social media platforms.
+ *
+ * Scope: #main-content .rw-sectioned-content section.rw-entity-social-media-links
+ */
+#main-content .rw-sectioned-content section.rw-entity-social-media-links ul {
+  /* Compensate the top margin of the li. */
+  /*margin: -16px 0 0 0;*/
+  padding: 0;
+  list-style: none;
+}
+#main-content .rw-sectioned-content section.rw-entity-social-media-links li {
+  /* To avoid borders on the next column. */
+  display: inline-block;
+  width: 100%;
+  margin: 16px 0 0 0;
+  padding: 12px;
+  border: 1px solid #e6ecef;
+}
+@media screen and (min-width: 768px) {
+  #main-content .rw-sectioned-content section.rw-entity-social-media-links ul {
+    column-count: 2;
+  }
+}
+@media screen and (min-width: 1024px) {
+  #main-content .rw-sectioned-content section.rw-entity-social-media-links ul {
+    column-count: 3;
+  }
+}
+
+
+/* External links. Display an icon after the link. */
```

Comment:
This is a valuable addition to the base theme

Reply:
This is also in https://github.com/UN-OCHA/rwint9-site/blob/70934976a3390afc50f202b4aa9c557aa2f026dc/html/themes/custom/common_design_subtheme/components/rw-useful-links/rw-useful-links.css#L47

Just a note to ourselves to remove this rule in both places when it's added to the base theme.